tools/c7n_azure/c7n_azure/session.py

```diff
@@ -28,7 +31,7 @@
 class Session(object):
 
     def __init__(self, subscription_id=None, authorization_file=None,
-                 resource=AZURE_PUBLIC_CLOUD.endpoints.active_directory_resource_id):
+                 resource='https://management.core.windows.net/'):
```

Comment:
This change makes sense but there is a constant already in constants.py for the storage resource name, so we can add this there too.